test-assets/test-scripts/DotnetCoreScriptWithNuget/main.csx

```diff
@@ -0,0 +1,5 @@
+#! "netcoreapp2.0"
+#r "nuget: Newtonsoft.Json,11.0.2"
+
+using Newtonsoft.Json;
+Console.WriteLine("Hello world!");
```

Comment:
It might be slightly better to write out one of the types from `Newtonsoft.Json` here ensuring that the assembly is correctly loaded